### ISSUE TITLE
Add accessibility check to docs pipeline

### ISSUE THREAD

**Issue by AaronCrawfis** [radius-project/docs#408]
Automatically check docs for accessibility issues and fail a PR build if any are found.

[AB#4835](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/4835)

**Comment by AaronCrawfis** [comment]
Won't address now. Please comment if there are accessibility issues that need to be addressed
